src/main/java/io/apicurio/datamodels/openapi/v3/io/Oas30DataModelReader.java

```diff
@@ -420,7 +421,7 @@ public void readRequestBody(Object json, Oas30RequestBody node) {
 
         node.$ref = $ref;
         node.description = description;
-        node.required = required;
+        node.required = required != null && required;
```

Comment:
When required is not set, it defaults to false, but maybe we don't want that so let me know if it should be reverted.